ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/dialogs/RestorePurchasesDialog.kt

```diff
@@ -129,10 +133,10 @@ private fun RestoringDialog(
 @Composable
 private fun PurchasesRecoveredDialog(
     localization: CustomerCenterConfigData.Localization,
-    onDismiss: () -> Unit,
+    onDismiss: (recovered: Boolean) -> Unit,
```

Comment:
This brought to mind some conversations we've had recently for paywalls, about whether to display our own restore dialogs in paywalls or leave it to the developer. While there are some cases we might want to display them from our side, currently we're letting the developer handle these dialogs on their side so they can customize any way they like. I think CustomerCenter doesn't require that much customization so it makes sense to keep it like this I think...